garage/tf/algos/npo.py

```diff
@@ -27,12 +26,13 @@ def __init__(self,
         self.optimizer = optimizer
         self.step_size = step_size
         self.name = name
-        super(NPO, self).__init__(**kwargs)
+        with tf.name_scope(name):
```

Comment:
this is confusing to read. i think it would be better to edit BatchPolopt to use the scope correctly.

Reply:
BatchPolopt only calls the init_opt of NPO and VPG, so I just moved this scope from the constructors to init_opt.